sensors/01-meteorological-data.Rmd

```diff
@@ -165,7 +153,7 @@ library(ggplot2)
 library(jsonlite)
 library(lubridate)
 
-weather_all <- fromJSON('https://terraref.ncsa.illinois.edu/clowder/api/geostreams/datapoints?stream_id=46431&since=2017-01-02&until=2017-01-31', flatten = FALSE)
+weather_all <- fromJSON('https://terraref.ncsa.illinois.edu/clowder/api/geostreams/datapoints?stream_id=46431&since=2017-01-01&until=2017-12-31', flatten = FALSE)
```

Comment:
```suggestion
weather_all <- fromJSON('https://terraref.ncsa.illinois.edu/clowder/api/geostreams/datapoints?stream_id=46431&since=2017-04-01&until=2017-08-01', flatten = FALSE)
```